helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
This file is created by helm boilerplate? Are there any manual changes in here as well?

Reply:
No manual changes, just boilerplate.